From 6cb20e0b298f41fe88984aebfe5454f936a0ae3a Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Fri, 20 Oct 2017 10:52:48 +0200
Subject: Disable CppunitTset_sc_*_functions_test for linux_aarch64 for now,
 too

> Oct 20 10:21:31 <sberg> erAck, when "Testing
>  file:///run/build/libreoffice/sc/qa/unit/data/functions/array/fods/linest.fods"
>  in CppunitTest_sc_array_functions_test fails with non-informative expected 1
>  vs. actual 0, how can I reasonably debug that?  (and no, I don't have an X11
>  connection to the relevant machine, to see what opening that .fods with LO
>  would look like there)
> Oct 20 10:25:04 <erAck> sberg: sigh.. we're still lacking an "analyze steps"
>  in the functions test loader, so far only loading the doc and inspecting
>  provides that; maybe getting individual cell results via UNO would work

So disable these tests for aarch64 for now too, to get Flathub builds unstuck,
until somebody finds a way to actually debug this.

Change-Id: I9bf30d0d7185497f9ad4b04b60b6a67f34a8c44e

diff --git a/sc/Module_sc.mk b/sc/Module_sc.mk
index bd5b769..0b600f7 100644
--- a/sc/Module_sc.mk
+++ b/sc/Module_sc.mk
@@ -69,7 +69,7 @@ $(eval $(call gb_Module_add_slowcheck_targets,sc, \
 # point weirdness (x87, registers, compiler optimization, ... whatever),
 # disable them until someone finds a real cure.
 
-ifneq ($(PLATFORMID),linux_x86)
+ifneq ($(filter-out linux_aarch64 linux_x86,$(PLATFORMID)),)
 $(eval $(call gb_Module_add_slowcheck_targets,sc, \
 	CppunitTest_sc_functions_test_old \
 	CppunitTest_sc_database_functions_test \
-- 
cgit v0.10.2

